main/src/library/LazyList.flix

```diff
@@ -607,6 +607,24 @@ namespace LazyList {
         case LList(xs)    => foldRightAcc(f, s, force xs, k)
     }
 
+    ///
+    /// Applies `f` to a start value `s` and all elements in `l` going from right to left.
+    ///
+    /// That is, the result is of the form: `f(x1, ...f(xn-1, f(xn, s))...)`.
+    ///
+    /// Forces the entire list `l`.
```

Comment:
It actually does not force the entire list. it only forces as long as needed by `f`. Agree?

Reply:
Oh that's right, sorry for this, I just forgot to update the comment